students/mariusz_michalczyk/lesson_02_flow_control/The_number_of_zeros.py

```diff
@@ -0,0 +1,10 @@
+size = int(input("How many numbers you want to enter: "))
+zeros = 0
+id = (size+1) - size
+while size:
```

Comment:
Typically, we write `for index in range(1, size + 1)`, also `id` is a built-in function that gives you the id of an object (typically memory pointer).